assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -110,6 +102,12 @@ function InspectorProvider({ children }) {
     }
   }, [currentPage]);
 
+  const ChecklistTabIcon = useCallback(() => {
+    return (
+      <ChecklistIcon checkpoint={currentCheckpoint} height={28} width={28} />
```

Comment:
morten wrote some docs about how we shouldn't resize icons: https://github.com/google/web-stories-wp/blob/main/docs/design-system.md#how-to-add-icons. Should we wrap this in a div that's 28x28px instead? The icons scale to the container size

Reply:
oh thank you! i'll fix this